Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinksSingleUser/O365DefenderSafeLinksSingleUser.yml

```diff
@@ -908,51 +908,51 @@ script:
           required: false
           isArray: false
         - name: page
-          description: Specifies the page number of the results you want to view. Valid input is a number between 1 and 1000. The default value is 1.
+          description: Sets the number of pages of results you want to view. Valid input is a number between 1 and 1000.
```

Comment:
@julieschwartz18 this is inaccurate. its is not the number of pages but the page number.